gjstest/internal/js/mock_instance.js

```diff
@@ -148,9 +148,26 @@ gjstest.internal.createMockOfPrototype_ =
   Tmp.prototype = prototype;
   var result = new Tmp;
 
+  // Object.getOwnPropertyNames does not include inherited methods. This method
+  // crawls up the inheritance chain and records all properties.
+  var inheritedPropertyNames = function(obj) {
```

Comment:
Does this need to be a local closure, or could you put it outside of this function to be clearer?